shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -54,7 +54,9 @@ CGPoint ConvertPointToGlobal(SemanticsObject* reference, CGPoint local_point) {
   // `rect` is in the physical pixel coordinate system. iOS expects the accessibility frame in
   // the logical pixel coordinate system. Therefore, we divide by the `scale` (pixel ratio) to
   // convert.
-  CGFloat scale = [[[reference bridge]->view() window] screen].scale;
+  UIScreen* screen = [[[reference bridge]->view() window] screen];
+  // Screen can be nil if the FlutterView is covered by another native view.
+  CGFloat scale = screen == nil ? 1 : screen.scale;
```

Comment:
Nice catch!  I'd say if you can't get the screen I'd use `[UIScreen mainScreen]` instead of using 1.